checkov/github/dal.py

```diff
@@ -158,6 +169,14 @@ def persist_branch_metadata(self) -> None:
         if branch_metadata:
             BaseVCSDAL.persist(path=self.github_conf_file_paths["branch_metadata"][0], conf=branch_metadata)
 
+    def persist_organization_metadata(self) -> None:
+        org_metadata = self.get_organization_metadata()
+        if org_metadata:
+            BaseVCSDAL.persist(path=self.github_conf_file_paths["org_metadata"][0], conf=org_metadata)
+
+    def persist_github_default_empty_file(self) -> None:
+        BaseVCSDAL.persist(path=self.github_conf_file_paths["default_github"][0], conf={CKV_GITHUB_DEFAULT: None})
```

Comment:
is there any problem just to set the `conf` as an empty `dict`?

Reply:
yes- in base_registry, if there is no entity (in this case all the file content is `{}`) , in line 111-112, so no result is returned for the check. 

that's why I chose to put something default for all checks that might need this. 
if we don't run them in the first place as they always pass- no need for this file :)